framework/components/ema-content-slider/ema-content-slider.scss

```diff
@@ -0,0 +1,28 @@
+.ema-content-slider {
+  margin: 0;
+}
+
+@media (min-width: map-get($ecl-grid-breakpoints, 'md')) {
+  .ema-content-slider {
+    display: grid;
+    grid-template-columns: 60% 40%;
+  }
+  .ema-content-slider .ecl-list-item {
+    align-items: center;
+    grid-column-start: 2;
+    &:not([aria-hidden='true']) {
+      display: flex;
+    }
+  }
```

Comment:
links in list-item should take 100% width (adding `flex-grow:1` to them should do the trick)

Reply:
you should not remove anything, but add flex-grow: 1 to links inside list-items